src/narjure/memory_management/persistence_manager.clj

```diff
@@ -22,18 +51,19 @@
   (shutdown!))
 
 (defn initialise
-  "Initialises actor:
-      registers actor and sets actor state"
+  "Initialises actor: registers actor and sets actor state"
   [aname actor-ref]
   (register! aname actor-ref)
-  (set-state! {:state 0}))
+  (set-state! {:path "d:/persistence/snapshot.nar" :concept-count 0 :received-states 0}))
```

Comment:
Hi, I already added environ it’s been committed to the PR.

 

From: Roman Volosovskyi [mailto:notifications@github.com] 
Sent: 30 April 2016 08:43
To: opennars/opennars2 <opennars2@noreply.github.com>
Cc: TonyLo1 <tony_lofthouse@btinternet.com>; Mention <mention@noreply.github.com>
Subject: Re: [opennars/opennars2] Actor update: concept-creator, forgettable-concept-collator, concept (#21)

 

In src/narjure/memory_management/persistence_manager.clj <https://github.com/opennars/opennars2/pull/21#discussion_r61666629> :

>    [aname actor-ref]
>    (register! aname actor-ref)
> -  (set-state! {:state 0}))
> +  (set-state! {:path "d:/persistence/snapshot.nar" :concept-count 0 :received-states 0}))

I don't mind if you will leave this task for me.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub <https://github.com/opennars/opennars2/pull/21/files/8bd9a8b32e14e24ffc289103d9c215ab1c3875fc#r61666629>